templates/repo/settings/branches.tmpl

```diff
@@ -37,19 +37,24 @@
 			</h4>
 
 			<div class="ui attached segment">
-				<div class="flex-list">
+				<div class="flex-list" id="protected-branches-list" data-update-priority-url="{{$.Repository.Link}}/settings/branches/priority">
 					{{range .ProtectedBranches}}
-						<div class="flex-item tw-items-center">
-							<div class="flex-item-main">
-								<div class="flex-item-title">
-									<div class="ui basic primary label">{{.RuleName}}</div>
+						<div data-id="{{.ID}}">
+							<div class="flex-item tw-items-center">
+								<div class="drag-handle tw-cursor-grab">
```

Comment:
It should be as simple as 
```
<div class="flex-item tw-items-center" data-id="{{.ID}} >
</div>
```

(I do not know why it's worth to add the extra wrapper)

Reply:
https://github.com/go-gitea/gitea/pull/32286/commits/f229044d28a6be7aee0f340bc6a270b4e0b80bf7